contracts/axelar-gateway/src/auth.rs

```diff
@@ -1,5 +1,5 @@
 use axelar_soroban_interfaces::types::{ProofSignature, ProofSigner, WeightedSigner};
```

Comment:
The PR title should use `test(axelar-gateway): ..` as the scope. See the conventional commit page
https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
got it. updated.